release notes/v0.9.0.md

```diff
@@ -0,0 +1,111 @@
+- First input delay ([FID](https://web.dev/fid/))
+- Cumulative layout shift ([CLS](https://web.dev/cls/))
+
+These measurements are now calculated for all tests without any additional work from your side. Simply run your test as you have been doing and you will be presented with the metrics in the output as you have used other measurements. This is the output after running the [examples/fillform.js](https://github.com/grafana/xk6-browser/blob/main/examples/fillform.js) script:
+
+```bash
+     webvital_cumulative_layout_shift..........: avg=0        min=0        med=0        max=0        p(90)=0        p(95)=0       
+     webvital_cumulative_layout_shift_good.....: 1       0.323332/s
+     webvital_first_contentful_paint...........: avg=278.86ms min=141.1ms  med=229.39ms max=466.1ms  p(90)=418.76ms p(95)=442.43ms
+     webvital_first_contentful_paint_good......: 3       0.969995/s
+     webvital_first_input_delay................: avg=300µs    min=200µs    med=300µs    max=399.99µs p(90)=379.99µs p(95)=389.99µs
+     webvital_first_input_delay_good...........: 2       0.646663/s
+     webvital_interaction_to_next_paint........: avg=16ms     min=16ms     med=16ms     max=16ms     p(90)=16ms     p(95)=16ms    
+     webvital_interaction_to_next_paint_good...: 1       0.323332/s
+     webvital_largest_content_paint............: avg=303.6ms  min=141.1ms  med=303.6ms  max=466.1ms  p(90)=433.6ms  p(95)=449.85ms
+     webvital_largest_content_paint_good.......: 2       0.646663/s
+     webvital_time_to_first_byte...............: avg=205.23ms min=104.79ms med=188.39ms max=322.5ms  p(90)=295.67ms p(95)=309.08ms
+     webvital_time_to_first_byte_good..........: 3       0.969995/s
+```
+
+You may have noticed other metrics in there too. we rely on the [web-vitals](https://github.com/GoogleChrome/web-vitals) JS library which exposes a few more metrics, so we've left them in for you to experiment with.
+
+You will no longer see `browser_first_contentful_paint` in the summary, and instead you can work with `webvital_first_contentful_paint`.
+
+### UX improvements and enhancements
+
+- [xk6-browser#788](https://github.com/grafana/xk6-browser/pull/788) Update the xk6-browser readme to highlight that it is now a module in k6.
+- [xk6-browser#803](https://github.com/grafana/xk6-browser/pull/803) Warns the user if the `browser.close` method is called more than once.
+- [xk6-browser#820](https://github.com/grafana/xk6-browser/pull/820) Add error handling to wildcard selectors, which cleans up the error output in the terminal.
+
+## Bug fixes
+
+- [xk6-browser#781](https://github.com/grafana/xk6-browser/pull/781) Fix mapping of `response` object's function from `jSON` to `json`.
+- [xk6-browser#779](https://github.com/grafana/xk6-browser/pull/779) Clear Zombie processes on panic.
+- [xk6-browser#817](https://github.com/grafana/xk6-browser/pull/817) Remove `--no-sandbox` as a default browser launch flag argument.
```

Comment:
I think this would be better placed in the `Maintenance and internal improvements` section.

Reply:
Yeah, it's a tricky one as to where this goes. Thinking about this a little more, could it be a better fit in breaking changes? The reason i say this is that if a user has been running under `root`, it will mean that xk6-browser will not be able to launch Chrome as root unless `--no-sandbox` is set. WDYT?

Reply:
Resolved in 532431a8ec95821d5ecbda5f67a87184c05d5b5c